docs/reference/release-notes/6.6.asciidoc

```diff
@@ -51,6 +51,34 @@ Also see <<breaking-changes-6.6,Breaking changes in 6.6>>.
 
 coming[6.6.0]
 
+[float]
+[[feature-6.6.0]]
+=== New features
+
+Machine Learning::
+* Store job configuration information in the new `.ml-config` index
+{pull}36698[#36698] (issue: {issue}32905[#32905])
+* Determine when data is missing from a bucket due to ingest latency
+{pull}35387[#35387] (issue: {issue}35131[#35131])
+
+[float]
+[[enhancement-6.6.0]]
+=== Enhancements
+
+Machine Learning::
+* Create the {ml} annotations index {pull}36731[#36731] (issue: {issue}33376[#33376])
+* Split in batches and migrate all jobs and datafeeds {pull}36716[#36716]
+(issue: {issue}32905[#32905])
```

Comment:
This one is really an implementation detail of the new feature "Store job configuration information in the new `.ml-config` index".  I'm not convinced it's worth listing separately.